secret_hitler/app.py

```diff
@@ -117,14 +117,18 @@ async def on_reaction_add(reaction, user):
         if len(game.votes) == len(game.players):
             if game.calculate_votes():
                 # Start Legislative Session
-                embed = discord.Embed(title="New Chancellor", description=chancellor.display_name + " is the new chancellor. Legislative Session starts now", color=discord.Color.dark_red())
+                embed = discord.Embed(title="New Chancellor",
+                                      description=game.chancellor.display_name +
+                                      " is the new chancellor. Legislative Session starts now",
+                                      color=discord.Color.dark_red())
```

Comment:
Here's one spot where I addressed a code error. Even if we don't want to keep all these changes, we should keep this one.